goal_src/engine/game/settings-h.gc

```diff
@@ -60,49 +60,6 @@
     )
   )
 
-(defmethod inspect setting-data ((obj setting-data))
```

Comment:
this one actually has some changes - the `movie` has a `ppoitner->process` in it.

Reply:
oh oops, i didn't see that.